fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryInUnionPlan.java

```diff
@@ -287,9 +308,9 @@ public InValues(@Nonnull String bindingName, @Nonnull final List<Object> values)
         @Override
         public int planHash(@Nonnull final PlanHashKind hashKind) {
```

Comment:
This is an incompatible change, isn't it?

Reply:
Yes, it is an incompatible change. I cannot mimic the old planners behavior.

Reply:
I updated the release doc. I hope that is sufficient. Again, the cascades planner does not stash anything into a global object that can serve as a stable counter.